docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: "3.5"
+services:
+  schema-registry:
+    image: confluentinc/cp-schema-registry:${IMAGE_TAG}
+    hostname: schema-registry
+    container_name: schema-registry
+    env_file: .env
+    depends_on:
```

Comment:
There is no broker?

Reply:
No, I tested it with the dev kafka cluster

Reply:
And it still works with depends_on? In that case I guess it is okay to leave it?